plugins/GSdx/resource.h

```diff
@@ -121,6 +121,7 @@
 #define IDC_OSDBUTTON                   2117
 #define IDC_OSD_MAX_LOG                 2118
 #define IDC_OSD_MAX_LOG_EDIT            2119
+#define IDC_OFFSETHACK_TEXT             2120
```

Comment:
In my humble opinion, this should be placed just below ``#define IDC_OFFSETHACK                  2043``.